### ISSUE TITLE
cron.archive_cleanup = DISABLED, but shown in the overview as active

### ISSUE THREAD

**Issue by somera** [go-gitea/gitea#12113]
- Gitea version (or commit ref): 1.12.1
- Git version: 2.20.1
- Operating system: Linux
- Database (use `[x]`):
  - [ ] PostgreSQL
  - [ ] MySQL
  - [ ] MSSQL
  - [x] SQLite
- Can you reproduce the bug at https://try.gitea.io:
  - [ ] Yes (provide example URL)
  - [ ] No
  - [x] Not relevant

## Description

This is my configuration entry for cron.archive_cleanup:

```
; Clean up old repository archives
[cron.archive_cleanup]
; Whether to enable the job
ENABLED = false
```

And in the monitoring overview I see:

old archives:
![image](https://user-images.githubusercontent.com/8334250/86280192-a04f3980-bbdb-11ea-9600-3f5ab6779c86.png)

all archives:
![image](https://user-images.githubusercontent.com/8334250/86280237-b826bd80-bbdb-11ea-88b9-76f3dce4013a.png)

**Comment by somera** [comment]
And when I start Gitea I sse in the log:

`2020/07/01 21:42:19 models/repo.go:1845:DeleteOldRepositoryArchives() [T] Finished: ArchiveCleanup`

**Comment by somera** [comment]
I think this was wrong log message which ist fixed in 1.12.2.
